cmds/wget/wget.go

```diff
@@ -48,13 +61,30 @@ func usage() {
 }
 
 func main() {
-	flag.Parse()
-	if flag.NArg() != 1 {
+	if flag.Parse(); flag.NArg() != 1 {
 		usage()
 	}
 
-	url := flag.Arg(0)
-	if err := wget(url, os.Stdout); err != nil {
-		log.Fatalf("%v\n", err)
+	argURL := flag.Arg(0)
+	if argURL == "" {
```

Comment:
I was surprised to see that this test is necessary but url.Parse with "" does not return an error!

Reply:
Is that a bug in url.Parse? Is it worth filing an issue?

Reply:
@rminnich I'm surprised too, but this is the actual url.Parse behaviour:
https://play.golang.org/p/cdd8V0-7oA
Probably it's worth filing an issue